### ISSUE TITLE
[QUESTION] Getting custom math masks/scale factors to work in GCHP

### ISSUE THREAD

**Issue by lfreese** [geoschem/GCHP#257]
I'm trying to set up a run in GCHP with country level scale factors that are math (cos) functions.

I followed the instructions shown here: https://hemco.readthedocs.io/en/latest/hco-ref-guide/more-examples.html?highlight=mask#scale-or-zero-emissions-with-a-rectangular-mask. But I'm having some issues with getting it to work. I took out the math functions so that it is just integer values as scale factors, and it still doesn't work. My impression is that its an error with how I have the scale factors/masks written in ExtData, and have changed formats around to match what the existing examples are and still can't get it to work. 

A second component of this is that I ideally want the scale factor to be a math function such as is shown under 'Scale Factors' here: https://geos-chem.readthedocs.io/en/latest/gcc-guide/03-config/hemco-config.html. I'm not sure if that is possible/feasible when using a scalefactor.txt file (it seems like it should be but I haven't seen an example of it before, and am still stuck just getting a normal scale factor to work first).

I'm using GCHP v13.4, attached my ExtData, HEMCO_Config, allPEs, scalefactors.txt, and slurm output below. I turned on all of the debug flags in HEMCO_Config and for allPEs.
[scalefactor.txt](https://github.com/geoschem/GCHP/files/9767781/scalefactor.txt)
[ExtData.txt](https://github.com/geoschem/GCHP/files/9767782/ExtData.txt)
[slurm-509412.txt](https://github.com/geoschem/GCHP/files/9767783/slurm-509412.txt)
[allPEs.txt](https://github.com/geoschem/GCHP/files/9767784/allPEs.txt)
[HEMCO_Config.txt](https://github.com/geoschem/GCHP/files/9767785/HEMCO_Config.txt)

**Comment by lizziel** [comment]
Hi @lfreese, do you also have the general run log (`gchp.log`) and the `HEMCO.log` files?

**Comment by lfreese** [comment]
Yes! Here is the gchp.log. There is no HEMCO.log (it didn't get that far).

[gchp.txt](https://github.com/geoschem/GCHP/files/9777822/gchp.txt)

**Comment by lizziel** [comment]
Hi @lfreese, I think what you are trying to do should not be a problem for GCHP since it is all related to HEMCO masking and scaling, something done separate from MAPL I/O. 

The main thing that pops out at me is you are inputting a text file. You need to make it a netcdf file.
```
#==============================================================================
# --- Country mask factor ---
#==============================================================================
9985 SCALE_COUNTRY /net/fs11/d0/emfreese/GCrundirs/IRFruns/stretch_cos/scalefactor.txt  - - - xy count 1
```

Also, a few tips for your debug runs. First, changing to a low resolution will make the run faster. Second, change this line in HEMCO_Config.rc:
`Logfile:                     HEMCO.log`
to this:
`Logfile:                     *`
This will send all HEMCO prints to the main log rather than HEMCO.log. It is default in 14.0 but not yet there in 13.4. It can make finding the problems in HEMCO easier.
